sdk/unstructured/yaml.go

```diff
@@ -9,6 +9,35 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+func YQFind(query string, content []byte) (bool, error) {
```

Comment:
`YQHasKey` maybe? I would expect a function called `YQFind` to fetch me the value for a key, not just a boolean of whether it exists or not.

Reply:
Done!